### DISCUSSION TITLE
French translation (and how to help ?)

### DISCUSSION THREAD

**Discussion by LydiaC** [pcottle/learnGitBranching#983] [General]
In one of the first three levels, it is written "partager c'est se soucier" or something similar
I would rather have said "partager c'est prendre soin des autres" or "c'est se soucier des autres"
But I would prefer some other French speakers to confirm and modify it, to be sure.

On another hand, can someone please explain me how to help for translation ?
I am rather unexperienced with github.
I found the language texts in the js files of each level in code section "learnGitBranching/src/levels", so I know what files to edit and how I am supposed to edit them.
But I don't know how to propose those files : am I supposed to do a pull request ? (and how does it work, once I clicked on "new pull request" how do I chose the proper "branches or forks above" ?) Or should I procede in another way ?
Thank you for your help

**Comment by pcottle** [comment]
Hello! Yeah for this project you'll need to submit a pull request of the changes you want. Here's a tutorial on how to do it with local edits:
https://www.youtube.com/watch?v=rgbCcBNZcdQ

but you could also just edit the individual file in github by clicking the pencil button here:
<img width="1214" alt="Screen Shot 2022-07-20 at 5 36 43 PM" src="https://user-images.githubusercontent.com/1135007/180100358-2d7cd1cd-6237-4f00-9871-8ff4c0a62f52.png">


and then submitting those changes. that process is far far easier if you just have one file to edit.

**Comment by LydiaC** [comment]
Hello,
I am sorry to ask, but at 3'10 the video shows a "git push origin nameOfTheFixBranch" command
I cloned the project, created a branch, did the modifications, commited on my branch, and tried to push (prior to ask for a pull request, right ?)
However the permission to push is denied.
Do you know why, should I procede with other branches or commands ?
Thank you for your explanation

**Reply by pcottle** [reply]
I would try https://stackoverflow.com/questions/19660744/git-push-permission-denied-public-key

As long as you have cloned the project and you pulled down the cloned version locally (that you own), you should have permission to push to your own repo.

**Reply by LydiaC** [reply]
Thanks a lot for your help finding this link
I solved it by declaring an ssh key on my github account and using the git remote rm origin / git remote add .. commands. I cloned the repo with http originally, so I tried with ssh instead.
I made the pull request. I hope it will help. Thank you for guiding me kindly through those steps.
